scout/server/blueprints/variant/templates/variant/variant.html

```diff
@@ -67,8 +69,8 @@
     {% endif %}
 
     <div class="row">
-      <div class="col-sm-12">{{ matching_variants() }}</div>
-      <div class="col-sm-4">{{ panel_basics() }}</div>
+      <div class="col-sm-12">{{ matching_variants(managed_variant) }}</div>
+      <div class="col-sm-4">{{ panel_classify(variant, institute, case, ACMG_OPTIONS, manual_rank_options, cancer_tier_options, dismiss_variant_options, mosaic_variant_options) }}</div>
```

Comment:
Great with the neat separation of classify and summary panels.

Looking at the variant page now it looks a bit odd, with that empty space under classify:

![image](https://user-images.githubusercontent.com/28093618/115835583-b81bd680-a416-11eb-96ca-c9299915cedc.png)

In comparison with the compact view we have at the moment:

![image](https://user-images.githubusercontent.com/28093618/115835669-d1bd1e00-a416-11eb-9cdf-10511ff576cd.png)

Could you try to rearrange the panels to use that space and to use all the page in general?